transaction/src/auth_hash.rs

```diff
@@ -110,6 +110,12 @@ impl TransactionPlan {
         for undelegation in self.undelegations() {
             state.update(undelegation.auth_hash().as_bytes());
         }
+        for validator_vote in self.validator_votes() {
+            state.update(validator_vote.auth_hash().as_bytes());
+        }
+        for _delegator_vote in self.delegator_vote_plans() {
+            // TODO: get the authorization hash of the delegator vote body for each plan
+        }
```

Comment:
I think we should merge this PR and then make another one to implement governance **actions** rather than merely action plans as implemented in this one? Or should I keep pushing on this and implement them here?